CHANGES.rst

```diff
@@ -18,6 +18,47 @@
 Changes
 =======
 
+`0.13.0 <https://github.com/SwissDataScienceCenter/renku-python/compare/v0.12.3...v0.13.0>`__ (2021-01-29)
+----------------------------------------------------------------------------------------------------------
+
+Bug Fixes
+~~~~~~~~~
+
+-  **core:** fix renku save with deleted files
+   (`#1849 <https://github.com/SwissDataScienceCenter/renku-python/issues/1849>`__)
+   (`93348f9 <https://github.com/SwissDataScienceCenter/renku-python/commit/93348f992eaf2b27b3eccf1460e1dd333d2fab77>`__)
+-  **core:** migration error when multiple outputs bind to the same input
+   (`#1832 <https://github.com/SwissDataScienceCenter/renku-python/issues/1832>`__)
+   (`bb19b47 <https://github.com/SwissDataScienceCenter/renku-python/commit/bb19b4786054f3e56ba64967a276906a393f6d74>`__)
+-  **core:** output git lfs error messages when there is an error
+   (`#1838 <https://github.com/SwissDataScienceCenter/renku-python/issues/1838>`__)
+   (`e2b5421 <https://github.com/SwissDataScienceCenter/renku-python/commit/e2b54215a60e7431d97e1f1e2e981f41008390c1>`__)
+-  **service:** reset cache after failed push
+   (`#1836 <https://github.com/SwissDataScienceCenter/renku-python/issues/1836>`__)
+   (`f41df17 <https://github.com/SwissDataScienceCenter/renku-python/commit/f41df17766c90d7ab43a21a1acea42a4333b9a81>`__)
+
+Features
+~~~~~~~~
+
+-  **cli:** add dataset show command
+   (`#1798 <https://github.com/SwissDataScienceCenter/renku-python/issues/1798>`__)
+   (`31e87f7 <https://github.com/SwissDataScienceCenter/renku-python/commit/31e87f7024bbdb96c685315a9ed6e87e9ae6a3a5>`__)
+-  **core:** cache for workflow migrations
+   (`#1853 <https://github.com/SwissDataScienceCenter/renku-python/issues/1853>`__)
+   (`f240d95 <https://github.com/SwissDataScienceCenter/renku-python/commit/f240d95d95475373046cc9199987fcdc095a9547>`__)
+-  **core:** new graph generation
```

Comment:
I think we should add a better description for this or leave it out, it's not very descriptive.

Reply:
agreed - you might spend a few sentences at the top of the release notes for this release describing this change since it's quite major.

Reply:
I prefer to leave it out since it's not still not visible to end users.